shuup/admin/modules/suppliers/views/edit.py

```diff
@@ -76,6 +78,25 @@ class SupplierEditView(SaveFormPartsMixin, FormPartsViewMixin, CreateOrUpdateVie
     base_form_part_classes = [SupplierBaseFormPart, SupplierContactAddressFormPart]
     form_part_class_provide_key = "admin_supplier_form_part"
 
+    def get_toolbar(self):
```

Comment:
Here would be more simple way to achieve this. But looks it should work. https://github.com/shuup/shuup/blob/master/shuup/admin/modules/contact_groups/views/edit.py#L33

Reply:
that code was copy-paste from HEAL custom delete model, will fix it